variants/motorgo_mini_1/pins_arduino.h

```diff
@@ -0,0 +1,79 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#include "soc/soc_caps.h"
+
+#define USB_VID 0x303A
+#define USB_PID 0x1001
+
+#define NUM_DIGITAL_PINS 6
+#define NUM_ANALOG_INPUTS 1
+
+#define analogInputToDigitalPin(p) \
+  (((p) < 20) ? (analogChannelToDigitalPin(p)) : -1)
+#define digitalPinToInterrupt(p) (((p) < 48) ? (p) : -1)
+#define digitalPinHasPWM(p) (p < 46)
```

Comment:
These can be deleted as they are globally defined.

```suggestion
```

Reply:
I included these based on [this PR](https://github.com/espressif/arduino-esp32/pull/9263), which says that they do need to be included for 2.x releases.

Are `NUM_DIGITAL_PINS` and `NUM_ANALOG INPUTS` used internally? I changed the values to match the number of pins exposed to users of our board, but I realize now that they might need to match the number available on the microcontroller.

Reply:
Ah yes, my bad. For 2.x it is still required.

Reply:
@SwapnilPande They are not used internally. You can leave it as it is :) This way most of boards are done in 2.X